Fix docs to make property listing in the website docs more useful. (#5746)
chef/inspec: docs-chef-io/content/inspec/resources/packages.md

      its('statuses') { should cmp 'installed' }
    end

## Properties

### statuses

> The `statuses` property tests if packages are installed on the system

Reviewer: ```suggestion
The `statuses` property tests if packages are installed on the system:
```

    its('statuses') { should cmp 'installed' }

### versions

> The `versions` property tests the versions of the packages installed on the system

Reviewer: ```suggestion
The `versions` property tests the package versions installed on the system:
```

    its('versions') { should cmp '3.4.0.2-4.el7' }

### architectures

> The `architectures` property tests the architecture of packages installed on the system

Reviewer: ```suggestion
The `architectures` property tests the architecture of packages installed on the system:
```
